src/mosm.sol

```diff
@@ -54,9 +54,21 @@ contract Mosm is LibNote {
     // Whitelisted feed readers, set by an auth
     mapping (bytes32 => mapping (address => uint256)) public bud;
 
+    address[] public lifted;
+    function lifted_length() public view returns(uint256) { return lifted.length; }
+
+    address[] public authedMedian;
+    function authedMedian_length() public view returns(uint256) { return authedMedian.length; }
+
+    address[] public authedOsm;
+    function authedOsm_length() public view returns(uint256) { return authedOsm.length; }
```

Comment:
I don't see any reasons why these functions should not be `external`. The arrays are never read inside the contract itself, so there is no reason to ever access the array's length from within the contract.

Making `external` functions instead of `public` functions should save us gas.

Reply:
updated